HelloTorchIOS/HelloTorchIOS/StrobeLights.swift

```diff
@@ -1,9 +1,39 @@
-//
-//  StrobeLights.swift
-//  HelloTorchIOS
-//
-//  Created by Luis Valle on 02/05/2018.
-//  Copyright © 2018 Novoda. All rights reserved.
-//
-
 import Foundation
+import AVFoundation
+
+class StrobeLights {
```

Comment:
Shouldn't this be renamed? Since it has nothing to do with the lights per se anymore, it just keeps a counter and timer right?